CONTENT/pitch.markdown

```diff
@@ -5,10 +5,10 @@
 
 ## Come, learn to code with us.
 
-Wizard Amigos Institute invites you to join its co-learning programming sessions in a friendly and playful atmosphere.
-Learn with us how to use state-of-the-art tools and methods to become a self-directed collaborative coder who co-creates the future using special magic programming wizardry.
+Wizard Amigos Institute is a programming school for children. We invite you to join our co-learning programming sessions in a friendly and playful atmosphere.
+Learn with us how to use state-of-the-art tools and methods. Become a self-directed collaborative programmer (=coder) and co-create the future using programming wizardry.
 
```

Comment:
Here same as above.
I changed "its" to "ours", because it is more neutral and doesnt mention "we vs. them" by saying "ours".
The same reason, why i wrote "the institute invites you" instead of "we".

`line 9:` i like the change you made there :-) cool +1